QoS/README.md

```diff
@@ -29,4 +29,11 @@ n/a
 
```

Comment:
The initial part of this conformance class should also include references to WMTS, not just WMS.

Reply:
1. The check is added for a request in which a LANGUAGE = unsupported-lang or this empty is specified. The response gets the ResponseLanguage as the DefaultLanguage

2. Reference to web map tiled services is added